test/fixtures/expectedformvitalsdata.json

```diff
@@ -6,14 +6,21 @@
     "formVitals": [
```

Comment:
in this data, the urls in cwv seem to be more than those in form vitals. is it expected?

Reply:
yes, form-vitals queries data on different checkpoints and cwv queries data on other set of checkpoints. Yes, it is expected